test/e2e/serial/tests/configuration.go

```diff
@@ -144,18 +144,18 @@ var _ = Describe("[serial][disruptive] numaresources configuration management",
 			Expect(err).ToNot(HaveOccurred(), "cannot get %q in the cluster", nroKey.String())
 			initialNroOperObj := nroOperObj.DeepCopy()
 
-			workers, err := depnodes.GetWorkers(fxt.DEnv())
-			Expect(err).ToNot(HaveOccurred())
-			if len(workers) < 2 {
-				e2efixture.Skipf(fxt, "the test requires at least 2 worker nodes, found %d", len(workers))
-			}
-
-			targetIdx, ok := e2efixture.PickNodeIndex(workers)
-			Expect(ok).To(BeTrue())
-			targetedNode := workers[targetIdx]
+			nodesNameSet := e2enrt.AccumulateNames(nrts)
+			// nrts > 1 is checked in the setup
+			targetedNodeName, _ := nodesNameSet.PopAny()
+			targetedNode := &corev1.Node{}
+			err = fxt.Client.Get(context.TODO(), client.ObjectKey{Name: targetedNodeName}, targetedNode)
```

Comment:
```
fxt.Client.Get(context.TODO(), client.ObjectKey{Name: targetedNodeName}, targetedNode).Should(Succeed(), "failed to pull node %s object", targetedNodeName)
```

Reply:
I prefered the 2 lines version specially in cases that the one line version turns out to be long (I have to scroll to the side to see the error message)

Reply:
lack in consistency in our codebase is a (minor) concern, but better to be addressed by a sweeping PR. Both versions are fine to me.

Reply:
ack